frontend/src/profile-form-FE/ReactForm.jsx

```diff
@@ -171,89 +178,102 @@ export default function ReactForm() {
         value="In Between"
       />
       <label className="form-input">In Between</label>
+      </label>
       <br />
 
       {/* Major */}
       <h1>Major: {major}</h1>
       {majorsList.map((majorItem) => (
         <div>
+           <label className="major-input">
           <input
             checked={major === majorItem}
             onChange={majorChange}
             type="radio"
             value={majorItem}
           />
           <label className="form-input">{majorItem}</label>
+          </label>
         </div>
       ))}
 
       {/* Romance */}
       <h1>Romantic Interests: {romance}</h1>
       {interestsList.map((interest) => (
         <div>
+          <label className="romance-input">
           <input
             onChange={romanceChange}
-            type="checkbox"
+            type="radio"
             placeholder={interest}
             value={interest}
           />
-          <span className="form-input">{interest}</span>
+          <label className="form-input">{interest}</label>
+          </label>
         </div>
       ))}
 
       {/* Friendship */}
       <h1>Friendship Interest: {friendship}</h1>
       {interestsList.map((interest) => (
         <div>
+          <label className="friend-input">
           <input
             onChange={friendshipChange}
-            type="checkbox"
+            type="radio"
             placeholder={interest}
             value={interest}
           />
-          <span className="form-input">{interest}</span>
+          <label className="form-input">{interest}</label>
+          </label>
         </div>
       ))}
 
       {/* Hobbies */}
       <h1>Hobbies: {hobbies}</h1>
       {hobbiesList.map((hobby) => (
         <div>
+          <label className="hobb-input">
           <input
             onChange={hobbiesChange}
-            type="checkbox"
+            type="radio"
             placeholder={hobby}
             value={hobby}
           />
-          <span className="form-input">{hobby}</span>
+          <label className="form-input">{hobby}</label>
+          </label>
         </div>
       ))}
 
       {/* Spirituality */}
       <h1>Spirituality: {spirituality}</h1>
       {religionsList.map((religion) => (
         <div>
+          <label className="spirit-input">
           <input
             checked={spirituality === religion}
             onChange={spiritualityChange}
             type="radio"
             value={religion}
           />
           <label className="form-input">{religion}</label>
+          </label>
         </div>
       ))}
 
       {/* Partying */}
       <h1>Party Favors: {partying}</h1>
       {recreationalsList.map((recreational) => (
         <div>
+          <label className="party-input">
           <input
             onChange={recreationalChange}
-            type="checkbox"
+            type="radio"

```

Comment:
I think this should be checkbox, not radio, to allow for multiple selection

Reply:
Correct! It should be a checkbox